release.py

```diff
@@ -129,7 +129,7 @@ def get_remote_name(github_user):
 
 def make_tag(bug_number, remote_name, tag_name, commits_since_tag):
     """Tags a release."""
-    message = "Tagged %s\n\n%s" % (tag_name, "\n".join(commits_since_tag))
```

Comment:
It seems weird to have the tag name in the tag message. I don't know why I did this originally.